### PR TITLE
Add LoRaWAN concentrator doc

### LINKED ISSUE #496: Document Lora Chips
#### Summary

SX1301 etc

#### Why do we need this ?

...

#### What is already there? What do you see now?

...

#### What is missing? What do you want to see?

...

#### How do you propose to document this?

...

#### Can you do this yourself and submit a Pull Request?

...

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Add LoRaWAN concentrator doc. Closes #496 

Also fixes homepage cards

#### Screenshots

Bildschirmfoto 2022-01-05 um 14 25 33

#### Changes

- Add concentrator doc
- Fix home page cards

#### Notes for Reviewers

...

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [ ] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/lorawan/concentrators/_index.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+Additionally, RAK Wireless provides a great [breakdown of available LoRa hardware](https://news.rakwireless.com/things-to-consider-when-picking-a-lorawan-gateway/).
+
+This page provides a quick description of each of the LoRa concentrators.
+
+## SX1301
+
+{{< figure src="SX1301.jpg" class="float plain" >}}
+
+The SX1301 is Semtech's first outdoor LoRaWAN concentrator, designed to provide worldwide LoRaWAN gateway capability in ISM bands. It is the first SoC to integrate Semtech's LoRa Concentrator IP.
```
Comment (jpmeijers) [inline doc/content/lorawan/concentrators/_index.md]: The SX1301 is not really a SoC as it does not contain a user programmable CPU/MCU. Does Semtech call it a SoC?

File: doc/content/lorawan/concentrators/_index.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+{{< figure src="SX1302.jpg" class="float plain" >}}
+
+The SX1302 succeeds the SX1301. It reduces current consumption, has better thermal capability, is cheaper, and can handle more traffic than its predecessor.
+
+- Up to -141 dBm sensitivity with SX1250 Tx/Rx front-end
+- 125 kHz LoRa reception with:
+    - 8 x 8 channels LoRa® packet detectors
+    - 8 x SF5-SF12 LoRa® demodulators
+    - 8 x SF5-SF10 LoRa® demodulators
+- 125 /250 / 500 kHz LoRa® demodulator
```
Comment (jpmeijers) [inline doc/content/lorawan/concentrators/_index.md]: 125 / 250 / 500 (missing space)

File: doc/content/lorawan/concentrators/_index.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+The SX1302 is succeeded by the SX1303.
+
+Read more about the SX1302 on Semtech's SX1302 [product page](https://www.semtech.com/products/wireless-rf/lora-core/sx1302).
+
+## SX1303
+
+{{< figure src="SX1303.jpg" class="float plain" >}}
+
+The SX1303 succeeds the SX1302, and is Semtech's most modern LoRa concentrator. It is pin and size compatible to its predecessor, and further reduces current, improves thermal capability, and is cheaper. In addition, it supports Time Difference On Arrival (TDOA) geolocation via a new Fine Timestamp capability.
```
Comment (jpmeijers) [inline doc/content/lorawan/concentrators/_index.md]: time difference OF arrival

File: doc/content/lorawan/concentrators/_index.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+- SX1255, SX1257 and SX1250
+- Single 32MHz clock
+
+Read more about the SX1303 on Semtech's SX1303 [product page](https://www.semtech.com/products/wireless-rf/lora-core/sx1303).
+
+## SX1308
+
+{{< figure src="SX1308.jpg" class="float plain" >}}
+
+The SX1308 is the lower powered concentrator compared to the SX1301/2/3. It is designed for indoor gateways.
```
Comment (jpmeijers) [inline doc/content/lorawan/concentrators/_index.md]: I would rather say `...is a lower power concentrator than the SX1301...`.

Is this however true, or is this what Semtech says? As far as I know the only difference between the SX1301 and the SX1308 is the noise figure.

File: doc/content/lorawan/concentrators/_index.md (hunk 1 above)
Comment (jpmeijers) [reply doc/content/lorawan/concentrators/_index.md]: How about "... is the first digital baseband chip to integrate Semtech's..."?

Comment (nejraselimovic) [inline doc/content/lorawan/concentrators/_index.md:22]:
```suggestion
The SX1301 is Semtech's first outdoor LoRaWAN concentrator, designed to provide worldwide LoRaWAN gateway capability in ISM bands. It is the first SoC to integrate Semtech's LoRa Concentrator IP.
```

**Pull request merged**